package/cfg/k3s-cis-1.24-hardened/master.yaml

```diff
@@ -131,7 +128,8 @@ groups:
                 value: "600"
         remediation: |
           Run the below command (based on the file location on your system) on the control plane node.
-          For example, chmod 600 <path/to/cni/files>
+          Note that for many CNIs, a lock file is created with permissions 750. This is expected and can be ignored.
```

Comment:
Do we want to say that by default K3s sets up the right permissions (600 and root:root), for Container Network Interface files maybe ? Then stipulate that if they are modified, one should apply the remediation  chmod 600 /var/lib/cni/networks/<filename> ?

Reply:
Sure I can reword